portfolio/src/main/webapp/index.html

```diff
@@ -9,6 +9,11 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 
+    <link rel="stylesheet" href="./css/master.css">
+
+    <!-- Freezeframe.js https://github.com/ctrl-freaks/freezeframe.js/tree/master/packages/freezeframe -->
+    <script src="https://unpkg.com/freezeframe/dist/freezeframe.min.js"></script>
+
     <script type="text/javascript" src="./js/script.js"></script>
     <script type="text/javascript" src="./js/activities.js"></script>
```

Comment:
From the google HTML/CSS style guide: "Omit type attributes for style sheets and scripts." https://google.github.io/styleguide/htmlcssguide.html#type_Attributes

Reply:
Done, thanks.